formats/detailed_guide/frontend/examples/detailed_guide.json

```diff
@@ -0,0 +1,71 @@
+    "change_note": "The second entry in the table Examples of salary sacrifice has been amended to correct the explanation of how much of the salary is subject to tax and National Insurance contributions.",
+    "first_published_at": "2014-06-12T10:00:00+01:00"
+  },
+  "links": {
+    "lead_organisations": [
+      {
+        "content_id": "6667cce2-e809-4e21-ae09-cb0bdc1ddda3",
+        "title": "HM Revenue & Customs",
+        "base_path": "/government/organisations/hm-revenue-customs",
+        "api_url": "https://www.gov.uk/api/organisations/hm-revenue-customs",
+        "web_url": "https://www.gov.uk/government/organisations/hm-revenue-customs",
+        "locale": "en",
+        "analytics_identifier": "D25"
+      }
+    ],
+    "worldwide_organisations": [
+      {
+        "content_id": "6667cce2-e809-4e21-ae09-cb0bdc1ddda3",
+        "title": "HM Revenue & Customs",
+        "base_path": "/government/world/organisations/hm-revenue-customs",
+        "api_url": "https://www.gov.uk/api/content/government/world/organisations/hm-revenue-customs",
+        "web_url": "https://www.gov.uk/government/world/organisations/hm-revenue-customs",
+        "locale": "en",
+        "analytics_identifier": "D25"
+      }
+    ],
+    "policy_areas": [
+      {
+        "content_id": "8034be95-4ac2-4fff-93c5-e7514ed9504a",
+        "title": "Tax and revenue",
+        "base_path": "/government/topics/tax-and-revenue",
+        "api_url": "https://www.gov.uk/api/content/government/world/topics/tax-and-revenue",
+        "web_url": "https://www.gov.uk/government/world/topics/tax-and-revenue",
+        "locale": "en"
+      }
+    ],
+    "parent": [
+      {
+        "content_id": "6e1d83bc-d37a-447c-bdac-31be4b441417",
+        "title": "PAYE",
```

Comment:
Me too! Here's the spelunking story:

```ruby
# whitehall rails c
> dg = Document.find_by_slug("salary-sacrifice-and-the-effects-on-paye").latest_edition
> dg.primary_specialist_sectors.first # only one
<SpecialistSector id: 47465, edition_id: 595175, tag: "business-tax/paye", created_at: "2016-02-18 09:02:19", updated_at: "2016-02-18 09:02:19", primary: true>
```

@tijmenb pointed out that `primary_specialist_sectors` map to `parent`:

![img_0561](https://cloud.githubusercontent.com/assets/1650875/14289996/39b881ea-fb55-11e5-97df-b98e84b67f6b.jpg)

And he also demonstrated how to find the above object in the `content-store`:

```ruby
# content-store rails c
> ci = ContentItem.find_by(base_path: "/topic/business-tax/paye"
> ci.title
"PAYE"
# etc
```